DB_Schema/create_leaderboard_tables.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE leaderboard_user(
```

Comment:
I think you should put this file in the same folder as your Python scripts. It is software rather than documentation. Thanks!

Reply:
Okay!